sentry_sdk/tracing.py

```diff
@@ -1317,7 +1320,9 @@ def continue_from_environ(
         **kwargs,  # type: Any
     ):
         # type: (...) -> POTelSpan
-        pass
+        # XXX actually propagate
+        span = POTelSpan(**kwargs)
+        return span
```

Comment:
This (and the stuff below) is here so that we get at least _some_ spans from the `continue_from_*` methods